app/package.json

```diff
@@ -54,6 +54,7 @@
     "fuse.js": "^6.6.2",
     "lodash-es": "^4.17.21",
     "ngx-clipboard": "^15.1.0",
+    "ngx-cookie-service": "^15.0.0",
```

Comment:
Curious why you chose to set the temporary token as a cookie vs adding it to the local storage.